src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -472,24 +473,30 @@ public function copyLanguage(
         }
 
         $document = $this->documentManager->find($uuid, $srcLocale);
-        $parentDocument = $this->inspector->getParent($document);
         if ($document instanceof ResourceSegmentBehavior) {
             $resourceLocatorStrategy = $this->resourceLocatorStrategyPool->getStrategyByWebspaceKey($webspaceKey);
         }
 
+        $parentUuid = null;
+        if (!$document instanceof ParentBehavior) {
```

Comment:
Isn't the `!` misplaced? I think this should only happen if it is of the type `ParentBehavior`...